neuraxle/hyperparams/distributions.py

```diff
@@ -23,12 +23,12 @@
 """
 
 import copy
-import math
 import random
 import sys
 from abc import abstractmethod, ABCMeta
 from typing import List
 
+import math
```

Comment:
How on earth did the autoformat put the math module below with external library imports such as numpy? Anyway...

Reply:
lol